repos/system_upgrade/el7toel8/actors/authselectapply/actor.py

```diff
@@ -0,0 +1,65 @@
+from leapp.actors import Actor
+from leapp.libraries.stdlib import call
+from leapp.models import Authselect, AuthselectDecision, Report
+from leapp.tags import IPUWorkflowTag, ApplicationsPhaseTag
+
+
+class AuthselectApply(Actor):
+    """
+    Apply changes suggested by AuthselectScanner.
+    
+    If confirmed by admin in AuthselectDecision, call suggested authselect
+    command to configure the system using this tool.
+    """
+    name = 'authselect_apply'
+    consumes = (Authselect, AuthselectDecision,)
+    produces = (Report,)
+    tags = (IPUWorkflowTag, ApplicationsPhaseTag)
+
+    def process(self):
+        model = next(self.consume(Authselect))
+        decision = next(self.consume(AuthselectDecision))
+
+        if not decision.confirmed:
+            return
+
+        if model.profile is None:
+            return
+
+        self.command = 'authselect select {0} {1} --force'.format(
+            model.profile,
+            ' '.join(model.features)
+        )
+
+        try:
+            call(self.command.split(' '))
+        except Exception as err:
+            self.failure(err)
+            return
+        except:
```

Comment:
Sorry, not getting it - what purpose has the second except block at L39?

Reply:
None. I removed it.